gdshowreelvote/vote/templates/vote/about.html

```diff
@@ -5,104 +5,68 @@
 {% endblock %}
 
 {% block content %}
-<h3>What is this website for?</h3>
 <p>
-The Godot Engine showreel website aims at providing a platform for members of the <a href="https://godotengine.org">Godot Engine</a> community to submit and rate videos for the regular showreels of the engine.
-A showreel is usually a 2 to 3-minutes-long video featuring a set of 10-15-seconds-long games (or tools) captures. See the previous showreels for examples of videos that suit the format: <a href="https://www.youtube.com/watch?v=UEDEIksGEjQ">2020 Desktop</a>, <a href="https://www.youtube.com/watch?v=AIapugketbs">2020 Mobile</a>, <a href="https://www.youtube.com/watch?v=NlKEO1N8wMM">2019 Desktop</a>, <a href="https://www.youtube.com/watch?v=ODn4oOqWGik">2019 Mobile</a>.
-It aims at illustrating the capabilities of the engine and offers some visibility to the featured games.
+    The Godot Engine showreel website is a platform to help with the submissions and voting of videos for the annual showreels. Each showreel is a 2 to 3-minutes-long video featuring a set of games or tools that hightlight the capabilities of the Godot Engine. You can see the full list of existing showreels on this <a href="https://www.youtube.com/playlist?list=PLeG_dAglpVo6EpaO9A1nkwJZOwrfiLdQ8">YouTube playlist</a>.
 </p>
 
 <p>
-To organise such showreels, a set of contributors usually took the time to gather the video proposals and rated each video one by one to select the best ones out of hundreds.
-However, doing this selection took us time, which was not used instead to improve the engine. That is why we created this website, so that our awesome Godot community could help us select the best and most impressive games for the engine showreels.
+    Users can submit their videos for one of the showreels (desktop, mobile, or tools), and maintainers will vote each entry. During the voting process, the scores will be hidden and videos will be presented in a random order to avoid any bias. After the voting phase ends, the video editors will use the most rated entries for the final showreels. The selection stages and deadlines are announced on our blog (https://godotengine.org/blog).
```

Comment:
```suggestion
    Users can submit their videos for one of the showreels (desktop, mobile, or tools), and maintainers will vote each entry. During the voting process, scores will be hidden and videos will be presented in a random order to avoid any bias. After the voting phase ends, the video editors will use the highest-rated entries for the final showreels. The selection stages and deadlines are announced on our blog (https://godotengine.org/blog).
```